src/circuits/runcircuit.jl

```diff
@@ -6,48 +6,52 @@
 --------------------------------------------------------------------------------
 """
 
-
-
 """
-    qubits(N::Int; mixed::Bool=false)
+    trivialstate(N::Int; mixed::Bool=false)
     
-    qubits(sites::Vector{<:Index}; mixed::Bool=false)
+    trivialstate(sites::Vector{<:Index}; mixed::Bool=false)
 
 
 Initialize qubits to:
 - An MPS wavefunction `|ψ⟩` if `mixed = false`
 - An MPO density matrix `ρ` if `mixed = true`
 """
-qubits(N::Int; mixed::Bool = false) =
-  qubits(siteinds("Qubit", N); mixed = mixed)
+trivialstate(N::Int; mixed::Bool = false) =
```

Comment:
Is the idea that we would eventually allow it to return an `LPDO` with a flag `lpdo = true`?

I also like the notation `trivialstate(LPDO, N; mixed = true)` for specifying you want an LPDO to be returned (it also helps with type stability, which is not much of a concern here).